tests/cpp/core/ConfigurationTests.cpp

```diff
@@ -78,6 +80,12 @@ void testBaseParameters(const Configuration& config)
 {
     BOOST_CHECK_EQUAL(config.getFullscreen(), true);
 
+    BOOST_CHECK_EQUAL(config.getBezelsVertical(),
+                      CONFIG_EXPECTED_BEZELS_VERTICAL);
+
+    BOOST_CHECK_EQUAL(config.getBezelsHorizontal(),
+                      CONFIG_EXPECTED_BEZELS_HORIZONAL);
```

Comment:
Since the #define is used only here, and the other checks in this function do not use #defines, I would just use the values here for simplicty